cli/ballerina-cli/src/main/resources/cli-help/internal-error-message.txt

```diff
@@ -1,9 +1,10 @@
 ballerina: Oh no, something really went wrong. Bad. Sad.
 
-There should be a file named "ballerina-internal.log" in the current directory.
-If you are able to share with us the code that broke Ballerina then
-we would REALLY appreciate if you would report this to us:
-go to https://github.com/ballerina-platform/ballerina-lang/issues and
-create a bug report with both this log file and the sample code.
+We appreciate it if you can report the code that broke Ballerina in 
+https://github.com/ballerina-platform/ballerina-lang/issues with the 
+`ballerina-internal.log` file in your current directory and your sample code.
```

Comment:
We don't create the `ballerina-internal.log` file now. Shall we reword it to say: `with this log and your sample code`

Reply:
Updated.